### DISCUSSION TITLE
How do I convert highlight and link style to cloze?

### DISCUSSION THREAD

**Discussion by iris2jane** [ObsidianToAnki/Obsidian_to_Anki#120] [Q&A] [answered]
<img width="521" alt="image" src="https://user-images.githubusercontent.com/36763930/101932970-7da75f00-3c16-11eb-903a-81859d9010f5.png">
Like 👆，thank you！

**Comment by Pseudonium** [comment, marked as answer]
Hmm, you could always wrap curly braces around them. But I'm assuming you would prefer not to have curly braces?

**Reply by iris2jane** [reply]
yeah……cause that would be too many files to be changed😂

**Reply by Pseudonium** [reply]
Hmm, well link style probably not because then you lose the functionality of actual links, but I might be able to implement an option in CurlyCloze to convert highlights to cloze deletions

**Reply by iris2jane** [reply]
That will be great. Thanks a lot！

**Reply by Pseudonium** [reply]
Check out v3.2.0 of the Obsidian plugin - I've added a feature that should suit your needs! Also expect a custom regexp for this to go up soon

**Reply by iris2jane** [reply]
Thank you very much! But I find that not each cards can be generated successfully when I press the plugin button，some paragraphs might be missed.

**Reply by Pseudonium** [reply]
Hmm, try opening an issue here with an example

**Reply by iris2jane** [reply]
<img width="722" alt="image" src="https://user-images.githubusercontent.com/36763930/102009943-c590c800-3d75-11eb-9d86-73d2bd5cbb25.png">

Compared to `{cloze}`, `==cloze==` is harder to be converted , it seems to be something wrong in 「highlight」👆

**Reply by Pseudonium** [reply]
Oh whoops, I think I know the error I made. I’ll see if I can push a fix soon

**Reply by Pseudonium** [reply]
Should be fixed now!
